### PR TITLE
More formatting fixes and code cleanup. Code behave exactly as before.

### PR DESCRIPTION
**Pull request by matshenricson**
A gazillion of changes, but mostly formatting and test fixes. If you are worried about them, looking at each commit is easier.

### CODE REVIEW

**Comment by matshenricson** [comment]
I have tripple checked these changes, but more eyes are welcome, of course.

**Comment by matshenricson** [comment]
Finally got all tests through. The tests are pretty flakey, I'd love to fix that at some time.

File: src/main/java/com/eternitywall/ots/BitcoinNode.java [hunk 1]
```diff
@@ -83,51 +82,58 @@ public BlockHeader getBlockHeader(Integer height) throws Exception {
...
+        String s = query.toString();
         URL url = new URL(urlString);
         Request request = new Request(url);
-        Map<String,String> headers = new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
         headers.put("Authorization", "Basic " + authString);
         request.setHeaders(headers);
         request.setData(s.getBytes());
-        Response response = null;  //sync call
+        Response response = null;  // sync call TODO: What?
```
Comment (RCasatta) [inline src/main/java/com/eternitywall/ots/BitcoinNode.java]: checked history, was in a try catch
no reason to keep the null init and the comment

File: src/main/java/com/eternitywall/ots/Timestamp.java [hunk 2]
```diff
@@ -1,53 +1,30 @@
...
 
 /**
- * Class representing com.eternitywall.ots.Timestamp interface
+ * Class representing a blockchain Timestamp.
  * Proof that one or more attestations commit to a message.
  * The proof is in the form of a tree, with each node being a message, and the
  * edges being operations acting on those messages. The leafs of the tree are
  * attestations that attest to the time that messages in the tree existed prior.
+ *
+ * @license LPGL3
```
Comment (RCasatta) [inline src/main/java/com/eternitywall/ots/Timestamp.java]: typo

File: src/main/java/com/eternitywall/ots/Timestamp.java [hunk 3]
```diff
@@ -73,141 +52,149 @@ public static Timestamp deserialize(byte[] ots, byte[] initialMsg) {
...
     /**
      * Create a Serialize object.
+     *
      * @param ctx - The stream serialization context.
      */
     public void serialize(StreamSerializationContext ctx) {
-
-        // sort
-        List<TimeAttestation> sortedAttestations = this.attestations;
+        // TODO: Not sure if this does what the author intended...
```
Comment (RCasatta) [inline src/main/java/com/eternitywall/ots/Timestamp.java]: fyi any doubt should be checked with the python reference implementation

File: src/main/java/com/eternitywall/ots/Timestamp.java [hunk 4]
```diff
@@ -73,141 +52,149 @@ public static Timestamp deserialize(byte[] ots, byte[] initialMsg) {
...
-        if (this.ops.isEmpty()) {
+
+        if (ops.isEmpty()) {
             ctx.writeByte((byte) 0x00);
+
             if (!sortedAttestations.isEmpty()) {
                 sortedAttestations.get(sortedAttestations.size() - 1).serialize(ctx);
             }
-        } else if (!this.ops.isEmpty()) {
+        } else if (!this.ops.isEmpty()) {   // TODO: Isn't this always true?
```
Comment (RCasatta) [inline src/main/java/com/eternitywall/ots/Timestamp.java]: yes, I really suppose so

File: src/main/java/com/eternitywall/ots/attestation/BitcoinBlockHeaderAttestation.java [hunk 5]
```diff
@@ -62,49 +61,51 @@ public static BitcoinBlockHeaderAttestation deserialize(StreamDeserializationCon
...
 
     @Override
-    public int hashCode(){
-        return Arrays.hashCode(this._TAG()) ^ this.height;
+    public int hashCode() {
+        return Arrays.hashCode(_TAG()) ^ height;
     }
 
     /*
      Verify attestation against a block header
```
Comment (RCasatta) [inline src/main/java/com/eternitywall/ots/attestation/BitcoinBlockHeaderAttestation.java]: missing javadoc style

File: src/main/java/com/eternitywall/ots/attestation/TimeAttestation.java [hunk 6]
```diff
@@ -1,40 +1,29 @@
...
-/** Class representing com.eternitywall.ots.Timestamp signature verification */
-public abstract class TimeAttestation  implements Comparable<TimeAttestation> {
-
-
-    private static Logger log = Utils.getLogger(TimeAttestation.class.getName());
 
+/**
+ * Class representing com.eternitywall.ots.Timestamp signature verification
+ *
+ * @license LPGL3
```
Comment (RCasatta) [inline src/main/java/com/eternitywall/ots/attestation/TimeAttestation.java]: typo

**Comment by RCasatta** [comment]
just a few nits comments, happy to merge this one which per my undestanding cover all https://github.com/opentimestamps/java-opentimestamps/pull/30 and more

**Comment by RCasatta** [comment]
closing cause #29 contain part of this

**Pull request closed without merging**